resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -30,7 +30,7 @@ private[spark] class PythonStep(
     val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
       "null"
     } else {
-      otherPyFiles.mkString(",")
+      KubernetesFileUtils.resolveSubmittedUris(otherPyFiles, filesDownloadPath).mkString(",")
```

Comment:
Hm, the remote files should most definitely be downloaded there. This was the case before this change.